pyomo/dae/tests/test_flatten.py

```diff
@@ -12,7 +12,92 @@
 from pyomo.environ import ConcreteModel, Block, Var, Reference, Set, Constraint
 from pyomo.dae import ContinuousSet
 # This inport will have to change when we decide where this should go...
```

Comment:
Typo: import

Reply:
Don't think this comment is relevant anymore, deleted.